deployer.go

```diff
@@ -349,6 +365,34 @@ func (d *deployer) performSequentialDeployment(sg serviceGroup) {
 			break
 		}
 
+		// read from etcd entry for app
+		//TODO context with timeout
+		etcdResp, err := d.etcdapi.Get(context.Background(), fmt.Sprintf("/ft/services/%s/healthcheck", getServiceName(u.Name)), nil)
+		if err != nil {
+			//TODO default to wait for unit
+			log.Printf("Error while getting etcd key for app from %s: %v", fmt.Sprintf("/ft/services/%s/healthcheck", getServiceName(u.Name)), err.Error())
+		}
+		log.Printf("HC node value: %s", etcdResp.Node.Value)
+		if etcdResp.Node.Value == "true" {
+			etcdResp, err := d.etcdapi.Get(context.Background(), "/ft/config/environment_tag", nil)
+			if err != nil {
+				//TODO default to wait for unit
+				log.Printf("Error while getting envrionment from %s: %v", "/ft/config/environment_tag", err.Error())
+			}
+			env := etcdResp.Node.Value
+			//TODO check gtg Endpoints
+			gtgResp, err := d.httpClient.Get(fmt.Sprintf("https://%s-up-coco.ft.com/__%s/__gtg", env, getServiceName(u.Name)))
```

Comment:
@martingartonft 

You were looking at an older commit, the actual PR goes to the machine, not the ELB. 

About going through vulcan, it's simple to do, and vulcan should be up all the time to direct me to the app. It does tie me to the __app format but I don't know about any plan to change that. 

Another point (albeit a weak one given the nr of call) is that etcd can barely handle the load as it is.